packages/terra-form/src/components/Textarea.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { PropTypes } from 'react';
```

Comment:
I debated that myself. I mainly stick to Textarea to keep the same naming convention of the html node textarea. I'm not picky on this though, and if you'd like it changed, I have no problem doing so.